src/components/navbar/SideBarWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Page from "./Page";
+import SideBar from "./SideBar";
+
+interface props {
```

Comment:
change `props` to `Props`

Reply:
คิดว่าใช้ `type` แทน `interface` ดีกว่า เพราะว่าอันนี้มันเป็น type อ่ะ ไม่ได้กำหนดว่าต้องมี methods อะไรบ้าง

Reply:
> คิดว่าใช้ `type` แทน `interface` ดีกว่า เพราะว่าอันนี้มันเป็น type อ่ะ ไม่ได้กำหนดว่าต้องมี methods อะไรบ้าง

I disagreed, it's common to use `interface` as a props definition. no matter if it has method or not.

Reply:
> change `props` to `Props`

better to use `SideBarWrapperProps` ?

Reply:
@intaniger Thanks for your advices ! เพิ่งรู้ว่ามี convention นี้ [link](https://levelup.gitconnected.com/mastering-typescript-a-guide-to-choosing-between-type-and-interface-c31d3527693b)

Reply:
@boomchanotai สวยพรี่สวย